xpath.js

```diff
@@ -4899,25 +4899,30 @@ var xpath = (typeof exports === 'undefined') ? {} : exports;
         return exports.select(e, doc, true);
     };
 
+    exports.isNodeLike = function (value) {
+        return value 
+            && typeof value.nodeType === "number" 
+            && Number.isInteger(value.nodeType)
+            && value.nodeType >= 1
+            && value.nodeType <= 11
+            && typeof value.nodeName === "string";
+    };
+
+    exports.isArrayOfNodes = function (value) {
+        return Array.isArray(value) && value.every(exports.isNodeLike);

```

Comment:
I think it's a little awkward to be accessing `isNodeLike` as a property of `exports` here and on line 4918.

Could you assign `isNodeLike` and `isArrayOfNodes` to `var`s, include them in the `assign()` call below, and access `isNodeLike` within this file as just `isNodeLike()`?

In the `assign()` call, please remember to include them as `isNodeLike: isNodeLike,` instead of `isNodeLike,`, since we are still maintaining ES5 compatibility for the moment.

Reply:
I was following the pattern of `select()` and `select1()`:

```js
exports.select1 = function (e, doc) {
    return exports.select(e, doc, true);
};
```

```js
exports.select = function (e, doc, single) {
    return exports.selectWithResolver(e, doc, null, single);
};
```

I'll gladly make whatever change you'd like though. Please confirm you'd like to proceed with your suggestion.

Reply:
I see, thank you for clarifying.

Yes, please go ahead and make the requested change for `isNodeLike` and `isArrayOfNodes`, but please leave `select1` and `select` as they are.